Airbrake/notifier/ABNotifier.m

```diff
@@ -527,6 +527,14 @@ + (NSData *)JSONString:(NSString *)filePath {
     NSError *error = NULL;
     NSError *jsonSerializationError = nil;
     NSString *dataStr = [[NSString alloc] initWithContentsOfFile:filePath encoding:NSUTF8StringEncoding error:&error];
+    
+    // chrash reports seems to be limited to 64kb, otherwise they are ignored by the API with the following error
+    // [Airbrake] {"code":400,"type":"Bad Request","message":"http: request body too large"}
+    // so we truncate the string to 60000 characters
+    if(dataStr.length > 60000) {
```

Comment:
1. Our API limit is 64KB, not 60000 characters, hence this is not the most precise check
2. Looks like you truncate a string. We need to check dumped JSON if it's bigger than 64KB and then truncate payload in the notice object